salt/states/pkgrepo.py

```diff
@@ -307,7 +370,15 @@ def managed(name, ppa=None, copr=None, **kwargs):
        :mod:`pkg.latest <salt.states.pkg.latest>` to trigger the
        running of ``apt-get update`` prior to attempting to install these
        packages. Setting a require in the pkg state will not work for this.
+
+    aptkey: Use the binary apt-key
```

Comment:
*question* would be appropriate to note that this is ignored if apt-key is not found on the path?